moxy.py

```diff
@@ -430,16 +511,18 @@ def encode_content(content: Union[str,list,dict]) -> Tuple[bytes, str]:
     `content` may be any of the following:
     - A file name string, in which case the content is loaded from the file and
       type inferred from the file's extension (json, js, html, xml, txt, md).
-    - A raw string, in which case it is encoded according as UTF-8, and the type
+    - A raw string, in which case it is encoded accordingly as UTF-8, and the type
       is inferred to be HTML if it starts with a `<`, otherwise JSON.
+    - A byte string, in which case it is assumed to be encoded as UTF-8, and the
+      type is inferred to be HTML if it starts with a `<`, otherwise JSON.
     - An object (such as a dictionary) that can be dumped as JSON, in which case
       it is converted to JSON.
     """
     content_type = "application/json"
     if isinstance(content, str):
         try:
             with open(content, "rb") as content_file:
-                if content.endswith("html"):
+                if content.endswith(".html"):
```

Comment:
The lack of `.` was actually intentional to cover `xhtml`, `shtml` and such variants, but TBH probably not much of an issue in practice, and `xhtml` might need its own type anyway

Reply:
Thanks for the context; I'll go ahead and revert this.